CHANGELOG.md

```diff
@@ -12,6 +12,21 @@ Mapbox welcomes participation and contributions from everyone.
 - Fixed an issue where a call to `MapboxNavigation#stopTripSession` would clear the routes reference and led to a `RoutesObserver` notification with empty routes collection. [#5685](https://github.com/mapbox/mapbox-navigation-android/pull/5685)
 - Fixed an issue where `AlternativeRouteMetadata` would get cleared after route refresh (whenever routes update reason was `RoutesExtra#ROUTES_UPDATE_REASON_REFRESH`). [#5691](https://github.com/mapbox/mapbox-navigation-android/pull/5691)
 
+## Mapbox Navigation SDK 2.4.0 - April 14, 2022
+
+#### Bug fixes and improvements
+- Fixed an issue where compressed tiles were incorrectly decode from the cache. [#5707](https://github.com/mapbox/mapbox-navigation-android/pull/5707)
```

Comment:
```suggestion
```suggestion
- Fixed an issue where compressed tiles were incorrectly decoded from the cache. This could have led to degraded enhanced map matching or unavailable resources. [#5707](https://github.com/mapbox/mapbox-navigation-android/pull/5707)
```
```